source/content/terminus/install.md

```diff
@@ -52,7 +51,7 @@ There are several ways to install Terminus, depending on your use case:
 1. Make the Terminus file executable. The example below assumes the same installation path as above:
 
   ```bash{promptUser: user}
-  chmod +X ~/.bin/terminus
+  chmod +x ~/.bin/terminus
```

Comment:
Also the text says `$HOME/bin` so maybe this should be `$HOME/bin/terminus` or at least `~/bin/terminus`

Reply:
Why?

Reply:
Invisible `.bin` is unusual, but I'm not going to stop you; the reader can reverse it if they have a different preference. As long as it's consistent.

Reply:
Unusual to who? Are there conventions on this? What's the latest study show?

`/me` uses .bin, fwiw.

Reply:
Unusual as in I have seen `$HOME/bin` N times, for a significantly large value of N, whereas I have seen `$HOME/.bin` one time. Also, if you google for `/home/.bin` you will find quite a few references to `bin` (still the minority in search results that are primarily about trash cans), but none to `.bin` in the first ten pages.

However, there is no RFC, so you may do as you like here.